internal/config/config.go

```diff
@@ -198,81 +127,6 @@ func getNamespaceFile(name string) string {
 	return path.Join(namespaceDirectory, name+".yaml")
 }
 
-func updateNamespaceToV2(header iofogctlNamespace) (iofogctlNamespace, error) {
```

Comment:
will re-add this functionality and add tests asap. Types have changed so this would no longer work as is